shell/src/main/java/alluxio/cli/fs/command/AbstractFileSystemCommand.java

```diff
@@ -17,7 +17,7 @@
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * The base class for all the FileSystem {@link Command} classes. It provides a place to hold the
+ * The base class for all the FileSystem {@link alluxio.cli.Command} classes. It provides a place to hold the
```

Comment:
this line is over 100 characters - you need to wrap it to the next line to pass the build

Reply:
When I wrap this line to the next line, I also get "test FAILed".